raijin_scripts/execute_fractional_cover/run

```diff
@@ -95,7 +92,13 @@ echo "**********************************************************************"
 echo ""
 echo "Executing fractional cover for ${YEAR} ${PRODUCT} with tag ${TAG}"
 ##################################################################################################
-# Run dea-submit-fractional cover process
+# Run datacube-fc submit two stage PBS job
 ##################################################################################################
 datacube-fc submit -v -v --project "${PROJECT}" --queue "${QUEUE}" --year "${YEAR}" --app-config "${APP_CONFIG}" --tag "${TAG}"
 EOF
+
+sleep 60s # Wait 60s before moving the submission log
```

Comment:
It strikes me as weird that we start up a `nohup` shell to launch the PBS submission in the background, but then wait 60 seconds and hope that it has run to completion.

Why not skip the `nohup` and run the whole script serially?